src/Scrabble/Move/Move.hs

```diff
@@ -42,9 +42,13 @@ createMove' :: Validator -- ^
 createMove' validate b (Rack rack) wp dict = if valid then go else errMsg where
   errMsg        = Left "error: rack missing input letters"
   rackLetters   = fmap letter rack
-  valid         = containsAll (toString putLetters) (toString rackLetters)
+  valid         = containsAllWithBlanks (toString putLetters) (toString rackLetters)
+  blanksUsed    = countBlanksUsed (toString putLetters) (toString rackLetters)
```

Comment:
I'll still change them since it will look way better. I'm more just curious if my mental model of laziness is correct